docs/en/source/API_walkthrough.rst

```diff
@@ -0,0 +1,147 @@
+.. list-table::  **API List** 
+   :widths: 5 30 15 35 15
+   :header-rows: 1
+
+   * - S.No.				
+     - External Interface 	 		 
+     - Module name
+     - Description
+     - Http verb
+     
+   * - 1.
+     - http://ip:port/ngsi-ld/v1/subscriptions						
+     - Subscription
+     - To add subscription in the NG Broker
```

Comment:
Please use Scorpio Broker as it is the official name now

Reply:
NGB is replaced by Scorpio Broker